### ISSUE TITLE
Error: runPath not found: test openfl/Export/linux64/cpp/bin/

### ISSUE THREAD

**Issue by jcward** [jcward/vscode-hxcpp-debug#56]
At some point, the output bin directory changed. It is now: `test openfl/Export/linux64/cpp/debug/bin/`

This may be related to OpenFL update (upgraded to 5.0), or hxcpp (upgraded from 3.4.69 to 3.4.90.) I changed back to hxcpp 3.4.69, and the error is unchanged, so I'm thinking maybe it's OpenFL. But it worked before I blew away my Export directory.

Ah, I believe I had the old executable sitting around from previous OpenFL versions, and that caused #55. Once I blew away my `Export` directory, it (rightly) started throwing this "runPath not found" error.

**Bottom line:** if you change OpenFL version, delete your Export directory.
